quirks/password-rules.json

```diff
@@ -623,6 +623,9 @@
     "id.sonyentertainmentnetwork.com": {
         "password-rules": "minlength: 8; maxlength: 30; required: lower, upper; required: digit; allowed: [-!@#^&*=+;:];"
     },
+    "id.westfield.com": {
+        "password-rules": "minlength: 9; maxlength: 20; required: lower; required: upper; required: digit; required: [!\"#&'()*,./:;?@[\\^_`{|}~];"
```

Comment:
There are two very special quirks of character classes (that I really need to add a linter to detect).

If `-` is present, it must be the first character in the class, and if `]` is present, it must be the last character.

Example:
`[- !]]`

Reply:
BUT it looks like `]` isn’t included in the code from the website, soooooo, this is good!